decidim-initiatives/spec/system/create_initiative_spec.rb

```diff
@@ -88,6 +88,30 @@
     end
   end
 
+  context "when rich text editor is enabled for participants" do
+    let(:initiative_type_minimum_committee_members) { 2 }
```

Comment:
This spec looks quite different than in the original PR (#10083).

Is there some reason for this?

Reply:
Yes, The original Spec file ( as seen on #10083) is a refactor that resulted from #9309, which is not backported. 

The spec has been initially migrated to 0.27 using commit 1cd70d933e85a1f480f4151b5796c32dc0d0c382, which failed as it could be seen in the below pipeline. https://github.com/decidim/decidim/actions/runs/4017451486/jobs/6901821161

in order to ensure a green pipeline during backport, i had to rewrite the test.

Reply:
Great, thanks!